net/ddns-scripts/files/usr/lib/ddns/update_porkbun_v3.sh

```diff
@@ -0,0 +1,165 @@
+#
+# Distributed under the terms of the GNU General Public License (GPL) version 2.0
+# 2024 Ansel Horn <dev@cahorn.net>
+#
+# Script for DDNS support via Porkbun's v3 API for the OpenWRT ddns-scripts package.
+#
+# Will will attempt to create a new or edit an existing A or AAAA record for the given
+# domain and subdomain. Existing CNAME and ALIAS records WILL NOT BE EDITED OR DELETED!
+# "username" and "password" configurations should be set to Porkbun API key and secret
+# key, respectively.
+#
+# Porkbun API documentation:
+# https://porkbun.com/api/json/v3/documentation#DNS%20Create%20Record
+#
+
+# Source JSON parser
+. /usr/share/libubox/jshn.sh
+
+# Define local variables
+local __API __DOMAIN_REGEX __DOMAIN __SUBDOMAIN __ADDR __TYPE __ID
```

Comment:
I know in other script `local` is also used for global variables, but this is not allowed for `ash`. This is not a function

Reply:
Got it - cleared and updated. :+1: